docs/tutorial/application-distribution.md

```diff
@@ -36,47 +50,43 @@ electron/resources/app
 └── index.html
 ```
 
-Then execute `Electron.app` (or `electron` on Linux, `electron.exe` on Windows),
+Then execute `Electron.app` (`electron` on Linux, `electron.exe` on Windows),
 and Electron will start as your app. The `electron` directory will then be
-your distribution to deliver to final users.
+your distribution to deliver to users.
 
-## Packaging Your App into a File
+### With asar archive
 
 Apart from shipping your app by copying all of its source files, you can also
-package your app into an [asar](https://github.com/electron/asar) archive to avoid
-exposing your app's source code to users.
+package your app into an [asar] archive to avoid exposing your app's source
+code to users.
```

Comment:
> to avoid exposing your app's source code to users.

I think we should avoid making this statement because it implies that asar prevents folks from being able to read an app's source code, which is definitely not true. Instead I would rewrite this to something like

```suggestion
package your app into an [asar] archive to improve the performance of reading files on platforms like Windows, if you are not already using a bundler such as Parcel or Webpack.
```